INN/umbrella-citylimits
# 47 homepage special projects section

## Changes

This pull request makes the following changes:

- Creates new City Limits special projects widget and adds styling
- Creates new `citylimits_get_series_posts` function based off of `largo_get_series_posts` to grab the needed series posts but also factors in featured in series and also allows you to specify how to order returned posts.

Desktop:
![screencapture-citylimits-test-2019-10-30-14_15_24](https://user-images.githubusercontent.com/18353636/67886437-d2c20780-fb1f-11e9-9ab8-62aebd56ace8.png)

Mobile:
![screencapture-citylimits-test-2019-10-30-14_15_36](https://user-images.githubusercontent.com/18353636/67886475-e1a8ba00-fb1f-11e9-9b71-4b54b60c8ea2.png)

## Why



For #47 

## Testing/Questions

Features that this PR affects:

- City Limits special projects widget


Questions that need to be answered before merging:

- [ ] Is there a better way to do any of this?
- [ ] The series images get kind of squashed when reducing the screen size due to having to set a hardcoded height for them to keep them all uniform, especially for series with no featured image. @benlk Do you have a better approach for this?
- [ ] Is the widget missing any options?

Steps to test this PR:

1. Add the new City Limits special projects widget to the homepage bottom widget area
2. Add title + select 3 series (with featured images)
3. Set excerpt to `# of Sentences` and `1` for `Excerpt Length`
4. Verify frontend looks ok and all posts are displaying in the correct order, including featured in series posts
5. Do steps 2-4 again but also test a series without a featured image
